tests/test_model.py

```diff
@@ -96,3 +94,13 @@ def test_predict(preprocess_data):
     roc_auc = ans['roc_auc']
     print("ROC_AUC: ", roc_auc)
     assert roc_auc <= MAX_ROC_AUC_SCORE
+
+
+def test_load_and_save():
+    model = modelgym.Model()
+    filepath = "data/newtree.yaml"
+    assert os.path.exists(filepath)
+    model.load_config(filepath)
```

Comment:
should also check that values from loaded model config (self.space, etc) match values from the fixure.